### DISCUSSION TITLE
Archiving multiple plots simultaneously

### DISCUSSION THREAD

**Discussion by vladaepro** [ericaltendorf/plotman#907] [Ideas]
I don't think this is possible with current version, but it would be useful in multiple plotters - single server configuration. For example, i have 5 plotters which all use plotman to archive plots to single (large) nvme on server. I can let plotman on server archive those plots further to hdds, but it will run only single rsync process so NVME 'buffer' can't handle so much input from 5 plotters and only single hdd output.

It would be nice to have possibility to set number of concurrent processes instead of single one, so we could chose to how many hdds to archive.

Someone will ask why just don't let every plotter archive to one of server's hdds. It works in ideal cases, when you have same number of HDD-s free as number of plotters (or more). But if you put 3 HDD-s, 5 plotters will saturate hdd-s throughput (writing 2 plots simultaneously on single hdd is slow).

**Comment by altendky** [comment]
Yes, it would be nice for plotman to support multiple simultaneous archive jobs.  There is no schedule for this happening.  In the mean time, have you looked at https://github.com/rjsears/chia_plot_manager?  I haven't looked at it in depth, but I know it provides more advanced plot management

**Reply by vladaepro** [reply]
Thanks, i take a look but it seems overcomplicated for my needs - i don't need plotting manager, just simple file distribution after plots are on single location. Think i'l write some bash script for that purpose, should be simpler :)
